.tekton/ansible-ai-connect-service-pull-request.yaml

```diff
@@ -240,7 +240,7 @@ spec:
       - name: TARGET_STAGE
         value: production
       - name: BUILD_ARGS
-        value: ["IMAGE_TAGS=pr-{{pull_request_number}} pr-{{pull_request_number}}.$(tasks.git-metadata.results.commit-timestamp)", "GIT_COMMIT=$(tasks.clone-repository.results.commit)"]
```

Comment:
we don't need to change PR workflow since it is not related

Reply:
see my notes in the PR explanation pls